scripts/train/train.py

```diff
@@ -545,14 +549,14 @@ def main(cfg: DictConfig) -> Trainer:
 
     eval_gauntlet_callback = None
 
-    if icl_tasks_config is not None:
```

Comment:
I think you should include the eval loader section above in the async run? It is a bit strange to only do some of the eval async.

Reply:
realizing that you did this because the eval.py script doesn't support a normal eval loader, only ICL. we should fix that along with adding support for it to async eval. doesn't necessarily need to be the same PR though.

Reply:
Can add as follow up! Do you think we want to run the eval loader at the same frequency as the icl tasks / gauntlet? I had assumed we may want to run the normal eval more frequently, and only larger eval tasks at major checkpoints

Reply:
yeah, right now `train.py` just runs all eval at the same interval. `eval.py` only runs eval once, so interval is not a relevant piece of this. So it would make sense to just keep the existing behavior of running at the same interval I think. We can add an optional interval arg to each individual ICL task if we want/it becomes worthwhile for some reason, but doesn't seem like a priority to me.

Reply:
Pushing as followup, assigned this to me https://databricks.atlassian.net/browse/RESEARCH-870